docker/conf/docker-compose-prod-deploy.yml.dist

```diff
@@ -0,0 +1,63 @@
+version: "3.1"
+services:
+    webserver:
+        restart: always
+        image: nginx:1.13-alpine
+        container_name: production-webserver
+        volumes:
+            - /var/www/production-content:/var/www/html/web/content
+            - ./web:/var/www/html/web
+            - nginx-conf:/etc/nginx/conf.d/
+        ports:
+            - 8000:8080
+        expose:
+            - 8000
+        links:
+            - php-fpm
+        networks:
+            - shopsys-network
+
+    php-fpm:
+        restart: always
+        image: production-php-fpm
+        container_name: production-php-fpm
+        volumes:
+            - /var/www/production-content:/var/www/html/web/content
+            - web-volume:/var/www/html/web
```

Comment:
Why is web folder volume different than other? Also I think that `/var/www/html/web/content` is now mounted twice with different volume type, one is mounted, second one is just volume.

Reply:
yes, first is named volume that is created with the same content that is in the php-fpm image and second is pointing via symlilnk on it.
Line 238 and lower.